internal/hud/renderer_test.go

```diff
@@ -133,25 +134,47 @@ ERROR: ImageBuild: executor failed running [/bin/sh -c go install github.com/win
 			},
 		},
 	}
-	rtf.run("all the data at once", 70, 20, v, vs)
+	rtf.run("all the data at once", 70, 20, v, plainVs)
 
 	v = view.View{
 		Resources: []view.Resource{
 			{
-				Name:                  "GlobalYAML",
-				CurrentBuildStartTime: ts,
-				LastBuildFinishTime:   ts,
-				LastBuildDuration:     1400 * time.Millisecond,
-				LastDeployTime:        ts,
-				LastBuildError:        "",
-				IsYAMLManifest:        true,
+				Name:                "GlobalYAML",
+				LastBuildFinishTime: ts,
+				LastBuildDuration:   1400 * time.Millisecond,
+				LastDeployTime:      ts,
+				LastBuildError:      "",
+				IsYAMLManifest:      true,
 			},
 		},
 	}
-	rtf.run("global yaml manifest", 70, 20, v, vs)
+	rtf.run("global yaml manifest", 70, 20, v, plainVs)
 
-	vs.AlertMessage = "this is only a test"
-	rtf.run("alert message", 70, 20, v, vs)
+	alertVs := plainVs
```

Comment:
fwiw, i've been playing around with a convention where each Test* method uses a common viewstate, which is kind of nice